redis/cluster.py

```diff
@@ -92,6 +92,23 @@ def fix_server(*args):
     return slots
 
 
+def parse_cluster_shards(resp, **options):
```

Comment:
What do you think about explaining why this is happening with a docstring, or some help text?